requirements/tests.txt

```diff
@@ -2,3 +2,4 @@ pytest>=3.5
 pytest-flake8
 pytest-cov
 pytest-black; platform_python_implementation == 'CPython'
+mypy>=0.930
```

Comment:
This is the first [mypy version](https://mypy-lang.blogspot.com/2021/12/mypy-0930-released.html) with support for `ParamSpec`.